torrt/trackers/rutracker.py

```diff
@@ -22,7 +22,7 @@ def get_id_from_link(self, url: str) -> str:
 
     def get_login_form_data(self, login: str, password: str) -> dict:
         """Returns a dictionary with data to be pushed to authorization form."""
-        return {'login_username': login, 'login_password': password, 'login': 'pushed', 'redirect': 'index.php'}
+        return {'login_username': login, 'login_password': password, 'login': '%E2%F5%EE%E4'}
```

Comment:
* Let's try to set `login` directly to `вход` value (`%E2%F5%EE%E4` is `win-1251`-urlencoded). Yet, it's strange they'd started to verify the value.
* Also `redirect` parameter should be just fine.

Reply:
I captured the login form data using Safari's inspector on a Mac. The login value is provided as-is, and no redirect field was present. Using "вход" in UTF-8 likely won't work if the backend expects CP1251, and hardcoding CP1251 into a Python script that defaults to UTF-8 doesn't seem ideal. Of course, utf8 вход can be converted into cp1251 before encoding into form but this would inflate code size for no good reason.

Reply:
> and no redirect field was present.

I suspect this can be related to a `referer` header. Anyway, if this param breaks nothing I propose leaving in intact.

> but this would inflate code size for no good reason.

Then we need at least a hint near that value: a hint about cp1251 `вход`.

Reply:
redirect is dead code these days, misleading that it is being used